### ISSUE TITLE
Playing a tone for more than 3 seconds causes issues

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt-adafruit#213]
Playing a tone for more than 3 seconds causes an integer overflow issue with codal, and you begin to experience weird behaviour. 

playTone for 3000 milliseconds, plays forever on the device
playTone for 5000 miliseconds, plays for a time shorter than a second.

Repro:
```
loops.forever(() => {
    music.playTone(262, 5000)
    music.playTone(Note.A, 1000)
})

```

**Comment by microsoftsam** [comment]
Codal issue: https://github.com/lancaster-university/codal-core/issues/1

**Comment by microsoftsam** [comment]
Fixed in codal-circuit-playground Snapshot 1.2.5

**Comment by abchatra** [comment]
Close this?
